requirements.txt

```diff
@@ -0,0 +1,31 @@
+# api generation
+namex>=0.0.8
+
+# formatting and linting
+black>=22
+flake8
+isort
+mypy
+
+# testing
+absl-py
+pytest
+pytest-cov
+numpy
+
+# pip build
+build
+
+# === runtime dependencies ===
+keras
+
+# Tensorflow.
+tensorflow-cpu~=2.17
```

Comment:
This won't work for GPU machines, so we might want to do a split between requirements-common, requirements-tf-gpu, etc. like for Keras

Reply:
Done